### ISSUE TITLE
Update lndclient branch `lnd-18-4` to track lnd `v0.18.4-beta` release

### ISSUE THREAD

**Issue by dstadulis** [lightninglabs/lndclient#194]
Updating `lndclient` branch `lnd-18-4` to point to the tagged version of `lnd` is a dependency for the custom channel saga and needs to be completed before merging [the mother of all PRs](link to the original PR).

To complete this task:

- [ ] 1. Tag / push `lnd v0.18.4-beta` to https://github.com/lightningnetwork/lnd/ tagged (at least an RC tag)
- [ ] 2. Update the `lndclient` branch `lnd-18-4` to point to the tagged version of `lnd`

This update is a prerequisite for the litd version (to be released with custom-channels code merged into main) [dependency chain](https://github.com/lightninglabs/lightning-terminal/pull/848).

**Comment by guggero** [comment]
Fixed by https://github.com/lightninglabs/lndclient/releases/tag/v0.18.4-6.
